install_config/install/disconnected_install.adoc

```diff
@@ -194,16 +200,26 @@ Replace `<tag>` with `{latest-int-tag}` for the latest version.
 endif::[]
 +
 ----
+# docker pull registry.access.redhat.com/openshift3/container-engine:<tag>
+# docker pull registry.access.redhat.com/openshift3/efs-provisioner:<tag>
+# docker pull registry.access.redhat.com/openshift3/jenkins-1-rhel7:<tag>
```

Comment:
The following jenkins-* images should belong to s2i builder images, but not "components for the additional centralized log aggregation and metrics aggregation components"